### ISSUE TITLE
Accessibility: empty cells missing role="gridcell" and aria-disabled="true"

### ISSUE THREAD

**Issue by ambrosew** [eureka2/ab-datepicker#55]
They should not be missing role="gridcell", and having aria-disabled="true" will tell Accessibility Insights tool to ignore color contrast requirements.

Before:
<td class="empty">31</td>

After
<td role="gridcell" aria-disabled="true" class="empty">31</td>
